utils/verification_test.go

```diff
@@ -120,27 +120,57 @@ func TestPackage2_2CanGetVerificationCode(t *testing.T) {
 		"File0": &spdx.File2_2{
 			FileName:           "file2.txt",
 			FileSPDXIdentifier: "File0",
-			FileChecksumSHA1:   "aaaaaaaaaabbbbbbbbbbccccccccccdddddddddd",
+			// FileChecksumSHA1:   "aaaaaaaaaabbbbbbbbbbccccccccccdddddddddd",
+			FileChecksums: []spdx.Checksum2_2{
+				spdx.Checksum2_2{
+					Algorithm: 1,
```

Comment:
"Algorithm" field should be populated using the defined constants (that is `SHA1`).
SHA1 in the current implementation is iota whose value is 0.